src/lib/comms/sol-coap.c

```diff
@@ -1438,6 +1457,19 @@ sol_coap_server_new(int port)
 }
 
 SOL_API int
+sol_coap_packet_get_buf(struct sol_coap_packet *pkt, uint8_t **buf, uint16_t *len)
```

Comment:
This shouldn't be exposed. The only user of this getter can afford to not call this, as discussed in the previous PR.